code/modules/mob/living/carbon/human/species.dm

```diff
@@ -226,7 +226,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	if(lastname)
 		randname += " [lastname]"
 	else
-		randname += " [pick(GLOB.last_names)]"
+		randname += " [pick(pick(GLOB.last_names_female), pick(GLOB.last_names_male))]"
```

Comment:
to powinno brać pod uwagę płeć

Reply:
też w przyszłośći może pokombinujcie zeby nie powtarzać tak kodu, np te pick(pick(...) można w jakiejś funkcji poprostu umieścić i będzie krócej i czytelniej

Reply:
Zaraz to poprawie, co do funkcji to też za chwilę ogarnę jakoś bo faktycznie przydałaby się